### PR TITLE
AniLibria striprussian and addrussian options

### PR DESCRIPTION
**Pull request by alikhanz**
https://github.com/Jackett/Jackett/issues/14019

### CODE REVIEW

**Comment by bakerboy448** [comment]
Won't this mess up groups? `-Group` => `-GroupRussian` ?

File: src/Jackett.Common/Definitions/anilibria.yml [hunk 1]
```diff
@@ -54,6 +62,13 @@ search:
       optional: true
     title:
       text: "{{ .Result.title_ru }} / {{ .Result.title_en }}{{ if .Result.title_alternative }} / {{ .Result.title_alternative }}{{ else }}{{ end }} ({{ .Result.year }}) [{{ .Result._quality }}]{{ if .Result._series }} - {{ .Result._series }}{{ else }}{{ end }}"
+      filters:
+        - name: re_replace
+          args: ["(\\([А-Яа-яЁё\\W]+\\))|(^[А-Яа-яЁё\\W\\d]+\\/ )|([а-яА-ЯЁё \\-]+,+)|([а-яА-ЯЁё]+)", "{{ if .Config.striprussian }}{{ else }}$1$2$3$4{{ end }}"]
+        - name: re_replace
+          args: ["[\\:\\-\\/\\|\\[\\]]", " "]
+        - name: append
+          args: "{{ if .Config.addrussian }} - RUSSIAN{{ else }}{{ end }}"
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/anilibria.yml:71]:
```suggestion
          args: "{{ if .Config.addrussian }} RUS {{ else }}{{ end }}"
```
it is usually only necessary to specify RUS to flag the language, and avoiding a dash prefix helps reduce the likelyhood of -groupname clashes.

**Review by garfield69 — CHANGES_REQUESTED** [review]

File: src/Jackett.Common/Definitions/anilibria.yml [hunk 2]
```diff
@@ -16,7 +16,15 @@ caps:
...
 
-settings: []
+settings:
+  - name: striprussian
+    type: checkbox
+    label: Strip Russian Letters
+    default: true
+  - name: addrussian
+    type: checkbox
+    label: Add RUSSIAN to end of all titles to improve language detection by Sonarr and Radarr. Will cause English-only results to be misidentified.
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/anilibria.yml:26]:
```suggestion
    label: Add RUS to end of all titles to improve language detection by Sonarr and Radarr. Will cause English-only results to be misidentified.
```

File: src/Jackett.Common/Definitions/anilibria.yml (hunk 1 above)
Comment (bakerboy448) [reply src/Jackett.Common/Definitions/anilibria.yml:71]: Still is the same issue tho, no?

some.title.year.quality-group => some.title.year.quality-groupRUS

Comment (garfield69) [reply src/Jackett.Common/Definitions/anilibria.yml:71]: there is a space in front of the RUS, does this not make a difference?

Comment (ilike2burnthing) [reply src/Jackett.Common/Definitions/anilibria.yml]: In the case of this indexer, we're building the title and there's no release group.

As for any similar future issue:

- `title-Group - RUS`, `title-Group RUS` and  `title-Group (RUS)` return `Group` as the release group
- `title-Group [RUS]` returns `RUS` as the release group
- `title - RUS` returns no release group.

**Comment by alikhanz** [comment]
> Won't this mess up groups? `-Group` => `-GroupRussian` ?

I have no issues with other indexers with same logic

**Pull request closed without merging**